assets/locales/fr.json

```diff
@@ -584,7 +584,15 @@
   "security": {
     "security": "Sécurité",
     "gatekeeper": "Etat Gatekeeper",
-    "sip": "Etat SIP"
+    "gatekeeper_active": "Gatekeeper Active",
```

Comment:
Please don't update the french localisation file unless you want to translate to french. Munkireport fills in the missing phrases with the default language (english). Leaving the phrases out of the localisation file makes it easier for the translators to spot missing items.